airbyte-integrations/connectors/source-stripe/metadata.yaml

```diff
@@ -5,7 +5,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: e094cb9a-26de-4645-8761-65c0c425d1de
-  dockerImageTag: 3.14.0
+  dockerImageTag: 3.15.0
```

Comment:
please remove the version pin on L17!

Reply:
I was thinking I'd do that in a separate PR, after testing this on our internal workspace using a LD version override.

Reply:
Got it, works for me!